x-pack/plugins/security_solution/server/lib/detection_engine/rules/prepackaged_rules/google_workspace_policy_modified.json

```diff
@@ -30,5 +30,5 @@
   ],
   "timestamp_override": "event.ingested",
   "type": "query",
-  "version": 9
+  "version": 11
```

Comment:
Unnecessary double version bump, was already bumped in previous `8.2 Rules` PR _without other content changes_: https://github.com/elastic/kibana/pull/129526/files#diff-7f967129fb57321097a06ac360f3ee2eb1da5e0acc07e0efabc947958fc356c2R33

Reply:
Double bump is the result of manual bumps done from the following.

https://github.com/elastic/detection-rules/pull/1870/files